Packs/PaloAltoNetworks_IoT/Playbooks/playbook-iot-with-servicenow-playbook.yml

```diff
@@ -0,0 +1,357 @@
+      model:
+        simple: ${PaloAltoNetworksIoT.device.model}
+      profile:
+        simple: ${PaloAltoNetworksIoT.device.profile}
+      raw_type:
+        simple: ${incident.rawType}
+      vendor:
+        simple: ${PaloAltoNetworksIoT.device.vendor}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 835
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "5":
+    id: "5"
+    taskid: 9b974236-9370-4e30-81a0-a3b860afe2a4
+    type: title
+    task:
+      id: 9b974236-9370-4e30-81a0-a3b860afe2a4
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 1535
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "6":
+    id: "6"
+    taskid: c8b4ea7e-efc0-410d-820b-e54f4b7e2bc3
+    type: condition
+    task:
+      id: c8b4ea7e-efc0-410d-820b-e54f4b7e2bc3
+      version: -1
+      name: IOT_CONFIG list configured?
+      type: condition
+      iscommand: false
+      brand: ""
+      description: 'to check whether the IOT_CONFIG is defined'
+    nexttasks:
+      '#default#':
+      - "9"
+      "yes":
+      - "4"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              simple: lists.IOT_CONFIG
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 660
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "7":
+    id: "7"
+    taskid: be71d420-9199-41ce-8cee-31f7db39a642
+    type: condition
+    task:
+      id: be71d420-9199-41ce-8cee-31f7db39a642
+      version: -1
+      name: ServiceNow Table Existed?
+      description: Check if a given value exists in the context. Will return 'no'
+        for empty empty arrays. To be used mostly with DQ and selectors.
+      scriptName: Exists
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "9"
+      "yes":
+      - "8"
+    scriptarguments:
+      value:
+        simple: ${PaloAltoNetworksIoT.raci.r_snow.table}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 1010
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "8":
+    id: "8"
+    taskid: aca86a67-dc23-4563-833f-1307866b85e1
+    type: regular
+    task:
+      id: aca86a67-dc23-4563-833f-1307866b85e1
+      version: -1
+      name: servicenow-create-record
```

Comment:
Better to add a conditional task to check if the integration is enabled before.

Reply:
If I have a step like this to merge two playbooks, what if I have the ServiceNow integration enabled for other things, but I do "not" want to create a record in this playbook?

Reply:
added the check